library/build.gradle

```diff
@@ -34,7 +34,8 @@ apply plugin: 'com.jfrog.bintray'
 apply plugin: "com.github.dcendents.android-maven"
 
 dependencies {
-    implementation "net.openid:appauth:${rootProject.appAuthVersion}"
+//    implementation "net.openid:appauth:${rootProject.appAuthVersion}"
+    compile(name: 'appauth-release', ext: 'aar')
```

Comment:
is this a binary in the repo?

Reply:
I see it now. What is in this lib, where did it come from?
In general i have a very big bias against adding artifacts to `lib` folders. I can can probably help suggest alternatives.

Reply:
That is a custom build of the AppAuth lib with "end of session" changes within it. I am planning to keep it only until Appauth merges my pull request.

Reply:
Lets chat about alternatives to checking in the jar before merging to master, as that doesn't track the changes.  We can link to your app-auth branch, make it a requirement to build that branch as a SNAPSHOT before building, shade it into project, etc.

Its fine for this feature branch (and maybe the upstream project will accept the changes before it becomes an issue?)